app/__init__.py

```diff
@@ -4,19 +4,20 @@
 import os
 from dotenv import load_dotenv
```

Comment:
Great work! We need to import load_dotenv to set up environment variables in our .env file